lua/harpoon/test/harpoon_spec.lua

```diff
@@ -41,14 +41,17 @@ describe("harpoon", function()
         }
 
         eq(expected, list.items)
+
+        vim.api.nvim_buf_delete(target_buf, { force = true })
```

Comment:
this should be cleaned up by the call to `be`.

utils should clear itself out and since these files were created via utils there should be bits keeping track there